packages/lit-html/src/lit-html.ts

```diff
@@ -443,7 +443,17 @@ const ELEMENT_PART = 6;
 const COMMENT_PART = 7;
 
 /**
- * The return type of the template tag functions.
+ * The return type of the template tag functions, {@linkcode html} and
+ * {@linkcode svg}.
+ *
+ * The `TemplateResult` object captures the arguments passed into the tag
+ * function. A special array of strings that retain their identity between
```

Comment:
I think this might be a little much inside detail for the docs on TemplateResult. It's good for a "How it Works" section though.

I'd try to simplify to something like:

A `TemplateResult` object holds all the information about a template expression required to render it: the template strings, expression values, and type of template (html or svg).

`TemplateResult` objects do not create any DOM on their own. To create or update DOM you need to render the `TemplateResult`. See [Rendering](https://lit.dev/docs/components/rendering) for more information.

I think I'd also include a link to render() and LitElement.render() so users navigating in an IDE don't have to jump to the website to see what can render these.